test/models/review_test.rb

```diff
@@ -167,14 +167,30 @@ class ReviewTest < ActiveSupport::TestCase
     assert_error @review, :plan_item_id, :invalid
   end
 
-  test 'validates required tag' do
+  test 'validates required business unit tag' do
     @review.taggings.clear
     @review.business_unit.business_unit_type.update! require_tag: true
 
     assert @review.invalid?
     assert_error @review, :taggings, :blank
   end
 
+  test 'validates required tags' do
+    skip if score_type != :manual
```

Comment:
Y acá como para romper un poco en algo casi sin importancia pondría:

```suggestion
    skip unless score_type == :manual
```

Reply:
Mañoso como nadie... xD

Reply:
El día que no haga un comentario sobre algo así, sabés que alguien está usurpando mi cuenta. Es una medida de seguridad =P